src/main/kotlin/no/nav/familie/ef/mottak/mapper/ArkiverDokumentRequestMapper.kt

```diff
@@ -24,12 +30,57 @@ object ArkiverDokumentRequestMapper {
                                       mapVedlegg(vedlegg, søknad.dokumenttype))
     }
 
+    fun fromEttersending(ettersending: Ettersending,
+                         vedlegg: List<EttersendingVedlegg>): ArkiverDokumentRequest {
+
+        val ettersendingDto = EttersendingMapper.toDto<EttersendingDto>(ettersending)
+
+        val hovedDokumentVarianter = lagHoveddokumentvarianterForEttersending(ettersendingDto, ettersending)
+
+        return ArkiverDokumentRequest(ettersendingDto.fnr,
+                                      false,
+                                      hovedDokumentVarianter,
+                                      mapEttersendingVedlegg(vedlegg,
+                                                             ettersendingDto.stønadType))
+    }
+
+    private fun lagHoveddokumentvarianterForEttersending(ettersendingDto: EttersendingDto,
+                                                         ettersending: Ettersending): List<Dokument> {
+        val tittel = "Ettersending til søknad om ${ettersendingDto.stønadType}"
+        val dokumenttype = utledDokumenttypeForEttersending(ettersendingDto)
+
+        val dokumentSomPdf = ettersending.ettersendingPdf?.let {
+            Dokument(it.bytes, Filtype.PDFA, null, tittel, dokumenttype)
+        } ?: error("Mangler forside for ettersendingen")
+
+        val dokumentSomJson = Dokument(ettersending.ettersendingJson.toByteArray(), Filtype.JSON, null, tittel, dokumenttype)
+
+        return listOf(dokumentSomPdf, dokumentSomJson)
+    }
+
+    private fun utledDokumenttypeForEttersending(ettersending: EttersendingDto) =
```

Comment:
Burde man kunne legge disse mappingene i `DokumenttypeUtil.kt` (kanskje trenger ett annet navn?) for å samle disse mappingene på ett sted